config/initializers/002-freedom_patches.rb

```diff
@@ -1,5 +1,11 @@
 # frozen_string_literal: true
 
-Dir["#{Rails.root}/lib/freedom_patches/*.rb"].each do |f|
-  require(f)
+RUN_WITHOUT_PREPARE = ["#{Rails.root}/lib/freedom_patches/rails_multisite.rb"]
```

Comment:
What was the problem that you ran into with the `rails_multisite` freedom patch?

Reply:
an error was coming from `initializers/200-first_middlewares.rb` and line
```ruby 
  Rails.configuration.middleware.unshift RailsMultisite::Middleware, RailsMultisite::DiscoursePatches.config
```

Problem is that `RailsMultisite::DiscoursePatches.config` is defined in the freedom patch and since the freedom patch is delayed with preload it was crashing.

https://github.com/discourse/discourse/blob/master/lib/freedom_patches/rails_multisite.rb#L35